cfme/tests/containers/test_smart_management.py

```diff
@@ -0,0 +1,105 @@
+from cfme.utils.wait import wait_for
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1),
+    pytest.mark.provider([ContainersProvider], scope='function')
+]
+
+TEST_ITEMS = [
+    pytest.mark.polarion('CMP-9948')(ContainersTestItem(
+        Container, 'CMP-9948', collection_obj="containers", keyword_name="name")),
+    pytest.mark.polarion('CMP-10320')(ContainersTestItem(
+        Template, 'CMP-10320', collection_obj="container_templates", keyword_name="name")),
+    pytest.mark.polarion('CMP-9992')(ContainersTestItem(
+        ImageRegistry, 'CMP-9992', collection_obj="container_image_registries",
+        keyword_name="host")),
+    pytest.mark.polarion('CMP-9964')(ContainersTestItem(
+        Node, 'CMP-9964', collection_obj="container_nodes", keyword_name="name")),
+    pytest.mark.polarion('CMP-9932')(ContainersTestItem(
+        Pod, 'CMP-9932', collection_obj="container_pods", keyword_name="name")),
+    pytest.mark.polarion('CMP-9870')(ContainersTestItem(
+        Project, 'CMP-9870', collection_obj="container_projects", keyword_name="name")),
+    pytest.mark.polarion('CMP-9854')(ContainersTestItem(
+        ContainersProvider, 'CMP-9854', collection_obj="containers_providers",
+        keyword_name="name"))
+]
+
+
+def get_entity(collection):
+    """Get a unique entity from the collection, randomly
+
+    Args:
+        collection: BaseCollection derived collection
+    """
+    # If there is at least one unique item
+    all_entities = collection.all()
+    names = [e.name for e in all_entities]
+    random.shuffle(all_entities)  # randomize what gets selected first
+    for entity in all_entities:
+        if [names.count(entity.name) == 1]:  # there's only single entity with this name
+            return entity
+        else:
+            logger.info('Skipping entity because its name is not unique: %s', entity)
+    else:
+        # ran out of entities to use
+        pytest.skip('Could not identify unique entity from collection for test: %s', collection)
+
+
+def get_clean_entity(collection=None, entity=None):
+    """Get an entity from collection, or used the passed one, and remove its tags
+
+    One of the two arguments needs to be passed. entity will take precedence if both passed
+
+    Args:
+        collection: BaseCollection derived collection
+        entity: BaseEntity derived collection
+    """
+    if collection is None and entity is None:
+        pytest.fail('Needed to pass a collection or entity within test')
+    else:
+        entity = entity or get_entity(collection)
+
+    tags = entity.get_tags()
+    entity.remove_tags(tags)
+    return entity
+
+
+@pytest.mark.parametrize('test_item', TEST_ITEMS,
+                         ids=[ContainersTestItem.get_pretty_id(ti) for ti in TEST_ITEMS])
+@pytest.mark.meta(blockers=[BZ(1609541)])
+def test_smart_management_add_tag(provider, appliance, test_item):
+
+    # validate no tag set to object
```

Comment:
@mshriver 
Can you please tell me, why don't we have ```polarion``` tags for this test case?

Reply:
@ganeshhubale good points, thanks!

This file is actually getting removed after some discussion with @jawatts. These tests are covered in other modules, and this ends up being a duplicate. 

It got added "back" basically, when I pulled these old commits back from the dead.